### PR TITLE
fix (comment)

### PR DESCRIPTION
**Pull request by sandymash**
- reset comment when closing without save

### CODE REVIEW

**Review by RoeiRom — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Review by yaniryos — APPROVED** [review]

**Review by yaniryos — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/CommentDialog/CommentDialog.tsx [hunk 1]
```diff
@@ -60,10 +60,11 @@ const CommentDialog = ({open, handleDialogClose}: Props) => {
                      step: 'Error occured in adding comment to investigation'
                  });
             })
-            .finally(onDialogClose);
+            .finally(handleDialogClose);
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/CommentDialog/CommentDialog.tsx:63]: Don't you need to use onDialogClose?

Comment (sandymash) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/CommentDialog/CommentDialog.tsx:63]: onDialogClose is for when the dialog is closed without saving

Comment (sandymash) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/CommentDialog/CommentDialog.tsx:63]: and so it resets the state

Comment (sandymash) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigationMenu/CommentDialog/CommentDialog.tsx:63]: when its saved the state doesnt need to reset

**Pull request merged**